.github/workflows/go.test.yml

```
@@ -63,7 +63,7 @@ jobs:
 
       - name: Test
         run: |
-          go install github.com/fatih/faillint || true
+          go install github.com/fatih/faillint@latest || true
```

Same here, let's not ignore errors.

```suggestion
          go install github.com/fatih/faillint@latest
```

---

fixed 👍